### PR TITLE
Update document for tensorrt yolo

### PR DESCRIPTION
**Pull request by wep21**
## Description

Update document for tensorrt yolo

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception/object_recognition/detection/tensorrt_yolo/README.md [hunk 1]
```diff
@@ -1,4 +1,126 @@
...
+### Node Parameters
+
+| Name                    | Type   | Default Value | Description                                                        |
+| ----------------------- | ------ | ------------- | ------------------------------------------------------------------ |
+| `onnx_file`             | string | ""            | The onnx file name for yolo model                                  |
+| `engine_file`           | string | ""            | The tensorrt engine file name for yolo model                       |
+| `label_file`            | string | ""            | The label file with label names for detected objects written on it |
+| `calib_image_directory` | string | ""            | The directory name including calibration images for int8 inference |
+| `calib_cache_file`      | string | ""            | The calibration cache file for int8 inference                      |
+| `mode`                  | string | "FP32"        | The inferece mode: "FP32", "FP16", "INT8"                          |
```
Comment (wep21) [inline perception/object_recognition/detection/tensorrt_yolo/README.md:119]:
```suggestion
| `mode`                  | string | "FP32"        | The inference mode: "FP32", "FP16", "INT8"                          |
```

File: perception/object_recognition/detection/tensorrt_yolo/README.md [hunk 2]
```diff
@@ -1,4 +1,126 @@
...
+### Node Parameters
+
+| Name                    | Type   | Default Value | Description                                                        |
+| ----------------------- | ------ | ------------- | ------------------------------------------------------------------ |
+| `onnx_file`             | string | ""            | The onnx file name for yolo model                                  |
+| `engine_file`           | string | ""            | The tensorrt engine file name for yolo model                       |
+| `label_file`            | string | ""            | The label file with label names for detected objects written on it |
+| `calib_image_directory` | string | ""            | The directory name including calibration images for int8 inference |
+| `calib_cache_file`      | string | ""            | The calibration cache file for int8 inference                      |
+| `mode`                  | string | "FP32"        | The inference mode: "FP32", "FP16", "INT8"                          |
```
Comment (wep21) [inline perception/object_recognition/detection/tensorrt_yolo/README.md:119]:
```suggestion
| `mode`                  | string | "FP32"        | The inference mode: "FP32", "FP16", "INT8"                         |
```

File: perception/object_recognition/detection/tensorrt_yolo/README.md [hunk 3]
```diff
@@ -1,4 +1,126 @@
...
+
+| Name       | Type                | Description     |
+| ---------- | ------------------- | --------------- |
+| `in/image` | `sensor_msgs/Image` | The input image |
+
+### Output
+
+| Name          | Type                                                  | Description                                        |
+| ------------- | ----------------------------------------------------- | -------------------------------------------------- |
+| `out/objects` | `autoware_perception_msgs/DetectedObjectsWithFeature` | The detetcted objects with 2D bounding boxes       |
```
Comment (wep21) [inline perception/object_recognition/detection/tensorrt_yolo/README.md:93]:
```suggestion
| `out/objects` | `autoware_perception_msgs/DetectedObjectsWithFeature` | The detected objects with 2D bounding boxes        |
```

File: perception/object_recognition/detection/tensorrt_yolo/README.md [hunk 4]
```diff
@@ -1,4 +1,126 @@
...
+
+This package detects 2D bounding boxes for target objects e.g., cars, trucks, bicycles, and pedestrians on a image based on YOLO(You only look once) model.
+
+## Inner-workings / Algorithms
+
+### Cite
+
+yolov3
+
+```bibtex
```
Comment (kmiya) [inline perception/object_recognition/detection/tensorrt_yolo/README.md:13]: IMHO: bibtex format is useful for researchers who write papers using TeX. For other people, the usual reference style might be easier to read. For example:

- Redmon, J., & Farhadi, A. (2018). Yolov3: An incremental improvement. arXiv preprint arXiv:1804.02767.
- Bochkovskiy, A., Wang, C. Y., & Liao, H. Y. M. (2020). Yolov4: Optimal speed and accuracy of object detection. arXiv preprint arXiv:2004.10934.
- Jocher, G., et al. (2021). ultralytics/yolov5: v6.0 - YOLOv5n 'Nano' models, Roboflow integration, TensorFlow export, OpenCV DNN support (v6.0). Zenodo. <https://doi.org/10.5281/zenodo.5563715>

Comment (kmiya) [reply perception/object_recognition/detection/tensorrt_yolo/README.md:13]: Anyway, [ultralytics/yolov5](https://github.com/ultralytics/yolov5) is licensed under GPL-3.0. Is that OK?

Comment (wep21) [reply perception/object_recognition/detection/tensorrt_yolo/README.md]: Addressed at https://github.com/tier4/AutowareArchitectureProposal.iv/pull/619/commits/3611fa7d486c8f7d83ca0cc003406b15aa42f07a

**Review by yukkysaito — APPROVED** [review]

**Comment by yukkysaito** [comment]
@wep21 precommitのところお願いします！

**Pull request merged**